Hardware/Hardware.xcodeproj/project.pbxproj

```diff
@@ -1422,9 +1426,17 @@
 /* End XCConfigurationList section */
 
 /* Begin XCSwiftPackageProductDependency section */
-		02B5147928254ED300750B71 /* Codegen */ = {
+		3F2B4AC32DDC30C300E5E49C /* XcodeTarget_Hardware */ = {
 			isa = XCSwiftPackageProductDependency;
-			productName = Codegen;
+			productName = XcodeTarget_Hardware;
+		};
+		3F2B4AC52DDC30D000E5E49C /* XcodeTarget_HardwareTests */ = {
+			isa = XCSwiftPackageProductDependency;
+			productName = XcodeTarget_HardwareTests;
+		};
+		3F2B4AC72DDC30D800E5E49C /* XcodeTarget_SampleReceiptPrinter */ = {
+			isa = XCSwiftPackageProductDependency;
+			productName = XcodeTarget_SampleReceiptPrinter;
```

Comment:
Side note: The SampleReceiptPrinter app might need to become part of the WooCommerce project when Hardware becomes a Swift package.

<img alt="image" src="https://github.com/user-attachments/assets/59600f0b-c6c4-4b62-b6df-d6fe3a732501" width=300/>